### ISSUE TITLE
suggestion on another use for stones and finding flint without gravel.

### ISSUE THREAD

**Issue by The-Lost-Echo** [wendall911/TinkerSurvival#34]
I'm playing with this mod and I'm enjoying the change to progression with one exception, I don't quite enjoy making gravel the first thing I have to find in order to get into the actual tinkering.

sometimes you just don't get any good* (not the ocean floor) gravel near your spawn, as is often the case in an island survival setting.  Even with gravel available, the progression feels...unfun, needing to find and sift gravel to get flint before you can begin to do anything else.

It would be better if there was a way to get past the "crude" age without ever touching a piece of gravel.  So my suggestion is to make stones serve another purpose by adding "rock testing".

 Basically it's the same as smacking flint on stone, you smack a rock against a stone and there is a chance it either breaks into 1-2 rock shards (ideally one you get from the part builder after crafting with extra material) or is revealed to actually be a piece of flint, which can then be struck again for the flint shards.

This feels like something that could be in-line with the current form of the mod, but removes something I feel is just extra work early on getting in the way of tinker's construct.

or, maybe I'm just lazy :(

**Comment by wendall911** [comment]
I think this is a really good idea. I'll bring it up with the visionary for the mod and see what he thinks and get back to you.

**Comment by wendall911** [comment]
Ok, adding this soon. Thank you for the suggestion

**Comment by wendall911** [comment]
This is now added ... finally ... to the 1.18.1 version. Thanks for the suggestion.
